Dockerfile

```diff
@@ -22,13 +21,18 @@ RUN apk --update add \
     rm /var/cache/apk/* && \
     ln -s /usr/bin/php7 /usr/bin/php
 
+COPY composer.json /usr/src/app/composer.json
```

Comment:
nice. I see this works, but tbh I don't totally understand it. the `COPY` directive knows to copy files relative to the Dockerfile into the workdir? `cp compser.* ./` wouldn't do anything, but I shouldn't think of `COPY` and `cp` as conceptually the same thing, huh?